src/modules/python/pbs_hooks/PBS_xeon_phi_provision.HK

```diff
@@ -0,0 +1,6 @@
+type=pbs
+enabled=false
+user=pbsadmin
+event=provision
+order=0
```

Comment:
I feel order should be 1 instead of order=0  because default value for  order should be 1 as per document.
Is there any specific reason we had kept order=0 for  pbs_hooks PBS_xeon_phi_provision ?